integration-tests/utils/integration-helpers.js

```diff
@@ -210,7 +210,12 @@ const withWorkspace = (test) => async (options) => {
     await test({ ...options, workspaceName });
   } finally {
     console.log('withWorkspace cleanup ...');
-    const didDelete = await withSignedInPage(deleteWorkspaceInUi)({ ...options, workspaceName });
+    let didDelete = false;
+    try {
+      didDelete = await withSignedInPage(deleteWorkspaceInUi)({ ...options, workspaceName });
+    } catch (err) {
+      console.error(`Error during workspace cleanup: ${err}`);
```

Comment:
If the test is still passing is `error()` correct here? Maybe it should be `warn()`?

Reply:
I don't feel strongly about this, but my thinking was: I want this to stand out for anyone looking at the logs, so the log-reader can easily know they might need to manually clean up workspaces.